broker-cli/utils/validations.js

```diff
@@ -137,11 +137,43 @@ function isBlockOrderId (str) {
   throw new Error('Block order IDs only contain upper and lower case letters, numbers, dashes (-) and underscores (_).')
 }
 
+/**
+ * Checks if a specified string is a valid date.
+ *
+ * @param  {String}  str - date
+ * @return {String}  str - date
+ * @throws {Error} If string cannot be parsed into a date
+ */
+function isDate (str) {
+  if (Date.parse(str)) {
```

Comment:
@martineehrlich JIC here is the link regarding our talk about how `Date.parse` might not give us the correct validation https://stackoverflow.com/questions/7445328/check-if-a-string-is-a-date-value/7445368#7445368